examples/flutter_gallery/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java

```diff
@@ -1,13 +1,13 @@
 package io.flutter.plugins;
 
 import io.flutter.plugin.common.PluginRegistry;
-import io.flutter.plugins.url_launcher.UrlLauncherPlugin;
+import io.flutter.plugins.urllauncher.UrlLauncherPlugin;
 
 /**
  * Generated file. Do not edit.
  */
 public final class GeneratedPluginRegistrant {
   public static void registerWith(PluginRegistry registry) {
-    UrlLauncherPlugin.registerWith(registry.registrarFor("io.flutter.plugins.url_launcher.UrlLauncherPlugin"));
+    UrlLauncherPlugin.registerWith(registry.registrarFor("io.flutter.plugins.urllauncher.UrlLauncherPlugin"));
   }
 }
```

Comment:
Yep.. this files is somehow being constantly regenerated without underscore in the name, I noticed and reverted it once, but missed it now it seems. Would be interesting to figure out why it takes underscore out.